packages/react-storage/src/components/StorageBrowser/views/LocationsView/LocationsView.tsx

```diff
@@ -118,17 +84,17 @@ export function LocationsView({
           className={`${CLASS_BASE}__locations-view-data-refresh`}
         />
         <Paginate
-          currentPage={currentPage}
-          disableNext={disableNext}
-          disablePrevious={disablePrevious}
-          handleNext={() => {
-            handlePaginateNext({ resultCount, hasNextToken });
-          }}
-          handlePrevious={handlePaginatePrevious}
+          currentPage={page}
+          disableNext={isPaginateNextDisabled}
+          disablePrevious={isPaginatePreviousDisabled}
+          handleNext={onPaginateNext}
+          handlePrevious={onPaginatePrevious}
         />
         <LocationsMessage />
         <Loading />
-        <DataTableControl onNavigate={onNavigate} range={range} />
+        {hasError ? null : (
```

Comment:
shouldn't we show an empty message in the table rather than rendering null?

Reply:
We'll show an error message but it looks like they're currently shown outside the table. I'm not sure if the new DataTable supports messages inside the table.

Reply:
I guess if you can't see the locations you're sort of out of luck for using the entire component. Maybe we could have a follow up to revisit the experience on this view, but not a blocker.